spdx-utils/src/main/kotlin/SpdxLicenseIdMapping.kt

```diff
@@ -25,7 +25,7 @@ import com.here.ort.spdx.SpdxLicense.*
  * A mapping from misspelled SPDX license IDs to valid SPDX IDs.
  */
 object SpdxLicenseIdMapping {
-    private val mapping = mapOf(
+    val mapping = mapOf(
```

Comment:
If you stripped `private` from here only for the tests, doesn't `internal` work here?

Reply:
I had this first, but then thought that we could also make it public as it is immutable, it might be useful to expose the mapping as part of the API. But I have no problem adding `internal` either.

Reply:
I just tried, `internal` doesn't work anyway for some reason.